src/plugins/template/src/config.cpp

```diff
@@ -85,6 +85,45 @@ Configuration::Configuration(const ov::AnyMap& config, const Configuration& defa
             model_priority = value.as<ov::hint::Priority>();
         } else if (ov::cache_encryption_callbacks == key) {
             encryption_callbacks = value.as<EncryptionCallbacks>();
+        } else if (ov::hint::scheduling_core_type == key) {
+            try {
+                schedulingCoreType = value.as<ov::hint::SchedulingCoreType>();
+            } catch (const std::exception&) {
+                OPENVINO_THROW("Wrong value ", value.as<ov::hint::SchedulingCoreType>(), "for property key ", key);
+            }
+        } else if (key == ov::hint::enable_hyper_threading.name()) {
```

Comment:
```suggestion
        } else if (key == ov::hint::enable_hyper_threading) {
```
property should be converted to name.